aries-site/src/examples/templates/filtering/FilteringWithSelect.js

```diff
@@ -33,322 +28,41 @@ const allData = [
-    filterResults = array.filter(item =>
-      // validates all filter criteria
-      filterKeys.every(key => {
-        // ignores non-function predicates
-        if (typeof criteria[key] !== 'function') return true;
-        return criteria[key](item[key]);
-      }),
-    );
-
-    if (searchValue) {
-      filterResults = filterResults.filter(o =>
-        Object.keys(o).some(
-          k =>
-            typeof o[k] === 'string' &&
-            o[k].toLowerCase().includes(searchValue.toLowerCase()),
-        ),
-      );
-    }
-    return filterResults;
-  };
-
-  useEffect(() => {
-    if (filters !== defaultFilters) {
-      setFiltering(true);
-    } else {
-      setFiltering(false);
-    }
-  }, [filters, setFiltering]);
-
-  return (
-    <Box
-      background="background"
-      gap="medium"
-      width={{ max: 'xxlarge' }}
-      pad={{ horizontal: 'medium' }}
-      margin="auto"
-      fill
-    >
-      <Header pad={{ top: 'medium' }}>
-        <Box gap="xsmall">
-          <Heading level={2} margin={{ bottom: 'small', top: 'none' }}>
-            Hosts
-          </Heading>
-          <Box align="center" direction="row" gap="small">
-            {!['xsmall', 'small'].includes(size) || searchFocused ? (
-              <Box width="medium">
-                <StyledTextInput
-                  ref={inputRef}
-                  type="search"
-                  icon={<Search id="search-icon" />}
-                  placeholder="Search"
-                  onBlur={() => setSearchFocused(false)}
-                  value={search}
-                  onChange={event => {
-                    setSearch(event.target.value);
-                    const nextData = filterData(
-                      allData,
-                      filters,
-                      event.target.value,
-                    );
-                    setData(nextData);
-                  }}
-                />
-              </Box>
-            ) : (
-              <Button
-                kind="toolbar"
-                id="search-button"
-                icon={<Search />}
-                onClick={() => setSearchFocused(true)}
-              />
-            )}
-            {(!['xsmall', 'small'].includes(size) || !searchFocused) && (
-              <Filters
-                data={data}
-                filtering={filtering}
-                setFiltering={setFiltering}
-                setData={setData}
-                filters={filters}
-                setFilters={setFilters}
-                filterData={filterData}
-                setSearch={setSearch}
-                // target is for demo purposes only, remove in production
-                target={containerRef && containerRef.current}
-              />
-            )}
-          </Box>
-
-          <RecordSummary data={data} filtering={filtering} />
-        </Box>
+export const FilteringWithSelect = () => (
+  <Page>
+    <PageContent gap="medium">
+      <Header>
```

Comment:
Will this have the Safari layout issue you experienced in the other PR from Header?

Reply:
Nope! The other PR issue was that `Grid` was the parent of both `Heading` and `Data`, so I moved `Grid` to just be the parent of `Data` to fix it.